teuthology/orchestra/console.py

```diff
@@ -266,8 +266,7 @@ def power_off_for_interval(self, interval=30):
         child = self._pexpect_spawn_ipmi('power on')
         child.expect('Chassis Power Control: Up/On', timeout=self.timeout)
         self._wait_for_login()
-        log.info('Power off for {i} seconds completed'.format(
-            s=self.shortname, i=interval))
+        log.info('Power off for {i} seconds completed'.format(i=interval))
```

Comment:
Same as above, should go to cleanup commit.

Reply:
I'm not sure maybe we need to leave 's', and add back it to the message?
Btw, since it is log, there can be better to use "%s" notation, like `log.info('some %s and %s', x, y)`

Reply:
same here. the current code is incorrect and `mypy` complains about it